src/signed_xml.ts

```diff
@@ -172,7 +181,8 @@ export class SignedXml implements XmlCore.IXmlSerializable {
                             found = findById(obj.GetXml()!, objectName!);
                             if (found) {
                                 let el = found.cloneNode(true) as Element;
-                                this.FixupNamespaceNodes(doc as Element, el, true);
+                                this.CopyNamespaces(el, el, true);
```

Comment:
@alexey-pelykh I updated this line to 
```ts
this.CopyNamespaces(found, el, true);
```
I've got `asic` namespace on verifying
```xml
<xades:SignedProperties xmlns:asic="http://uri.etsi.org/02918/v1.2.1#" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" xmlns:xades="http://uri.etsi.org/01903/v1.3.2#" Id="xades-id-a46287b499d8">
```

Could you try to update and test signing again?

Reply:
That would work, done